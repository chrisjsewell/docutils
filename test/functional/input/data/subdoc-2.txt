================
 Sub-Document 2
================

This is another document intended to be included as a sub-document.
