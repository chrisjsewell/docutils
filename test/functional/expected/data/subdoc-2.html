<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.5: http://docutils.sourceforge.net/" />
<title>2.14.9.1.1&nbsp;&nbsp;&nbsp;Sub-Document 2</title>
<link rel="stylesheet" href="../../../docutils/writers/html4css1/html4css1.css" type="text/css" />
</head>
<body>
<div class="document">


<div class="section" id="sub-document-2">
<h1><a class="toc-backref" href="../standalone_rst_html4css1.html#id85">2.14.9.1.1&nbsp;&nbsp;&nbsp;Sub-Document 2</a></h1>
<p>This is another document intended to be included as a sub-document.</p>
</div>
</div>
</body>
</html>
